canvas_oauth/models.py

```diff
@@ -45,3 +45,10 @@ def expires_within(self, delta):
             return False
 
         return self.expires - timezone.now() <= delta
+
+    def __str__(self):
+        return "CanvasOAuth2Token:%s" % self.user
```

Comment:
Per our offline discussion, I think maybe we hold off updating to f-strings for now. I found two instances where I introduced f-strings, and I'll revert for now to stay consistent with the existing style. This will also retain python3.5 compatibility a little bit longer.